netmaster/daemon/daemon.go

```diff
@@ -246,28 +260,44 @@ func (d *MasterDaemon) runLeader() {
 	// Create HTTP server and listener
 	server := &http.Server{Handler: router}
 	server.SetKeepAlivesEnabled(false)
-	listener, err := net.Listen("tcp", d.ListenURL)
+
+	ctrlListener, err = net.Listen("tcp", d.ControlURL)
```

Comment:
We need to handle the scenario where controlURL = "192.1.1.1:9999" and listernURL = ":9999".